deploy/scripts/src/mobile/utils/test-client.ts

```diff
@@ -0,0 +1,23 @@
+import http, { type Response } from 'k6/http'
+import { getUrl } from './url'
+import { config } from './config'
+
+export function postTestClientStart (): Response {
+  return http.post(
+    getUrl('start', config.testClientExecuteUrl),
+    JSON.stringify({ frontendUri: config.frontendUrl }),
```

Comment:
Note: We have removed `target` from the request body here due to issues in the test client configuration which we've discussed before. We ran the tests locally against build to ensure there were no issues with running it against that environment without specifying the target here.

Reply:
not quite in answer to your point, but what's the rationale for needing a frontendUri? I thought that was only needed when using the default test client with your own FE stack

Reply:
I think if you omit frontendUri then the WebLocation will use the AWS resource ID instead of the pretty URL. Seems better to use the pretty URL if we can, what do you think? (Hence why I was saying we would have provided the `target` as well, had it not been for the issue with the test client)

Reply:
ah that makes sense - we should really look at updating the test client!